src/main/fc/rc_controls.c

```diff
@@ -118,11 +119,20 @@ throttleStatus_e calculateThrottleStatus(void)
     return THROTTLE_HIGH;
 }
 
+#define ARM_DELAY_TIME 500000 //arm,disarm delay usec
```

Comment:
Should be `ARM_DELAY_TIME_US`.

As a rule of thumb, if something needs a comment, instead of adding the comment, check if it can be rewritten in a way that eliminates the need for a comment.